Implement mongo connections to display all data in the front end related to a searched docket

adams-k opened the pull request.

bjcoleman commented:
We are going to add LinkedIn profile links - but I want to use this as an in-class exercise.  Please leave this open for now.

bjcoleman commented:
@anna-huang17 @vitoleone27 Your format is not correct.  You need to use [Markdown links](https://www.markdownguide.org/basic-syntax/#links) as the formatting.  Please update.

bjcoleman commented:
Great work, everyone!  A PR with 18 contributors!  :)

adams-k committed:
Implement getting comment and document data related to searched docket

adams-k committed:
Saving changes

adams-k committed:
Fix dates displayed on front end

adams-k committed:
Merge branch 'main' of https://github.com/cs334s24/Mirrulations_Search into db_data

adams-k committed:
Verify agency ID works with frontend

adams-k committed:
Refactor methods for testing

Commit:
Address issues in mock database

Added necessary fields in the mock database for
Mongo in order to pass the tests

Commit:
Created mock for Cursor object

Implemented a mock for PyMongo's Cursor object to
make the tests compliant.

Also changed metric for testing to exclude the mock
Python modules.

Commit:
Merge branch 'main' of https://github.com/cs334s24/Mirrulations_Search into db_data

adams-k committed:
Made compliant with pytest coverage

xtelos commented:
Looks good, merging
